extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -289,24 +289,24 @@ public function executeBackgroundImageUpload() {
 	 * @param UploadBase $upload
 	 * @return array
 	 */
-	private function uploadImage($upload) {
+	private function uploadImage( $upload ) {
 		global $wgRequest, $wgUser, $wgEnableUploads;
 
-		$uploadStatus = array("status" => "error");
+		$uploadStatus = [ "status" => "error" ];
 
-		if ( empty( $wgEnableUploads )) {
+		if ( empty( $wgEnableUploads ) ) {
 			$uploadStatus["errors"] = [ wfMessage( 'themedesigner-upload-disabled' )->plain() ];
 		} else {
 			$upload->initializeFromRequest( $wgRequest );
 			$permErrors = $upload->verifyPermissions( $wgUser );
 
 			if ( $permErrors !== true ) {
-				$uploadStatus["errors"] = array( wfMsg( 'badaccess' ) );
+				$uploadStatus["errors"] = [ wfMsg( 'badaccess' ) ];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)